server/controllers/routes/admin/cohort/addCohort.js

```diff
@@ -0,0 +1,22 @@
+const {
+  postCohort,
+} = require('../../../../database/queries/cohort/postCohort');
+
+const addCohort = async (req, res, next) => {
+  try {
+    const { rows } = await postCohort(req.body);
+    return res.json({
+      StatusCode: 201,
+      data: {
+        cohortId: rows[0].id,
+        message: 'Cohort added successfully',
```

Comment:
@rehabas It might be helpful to return all of rows[0] as data to the frontend here, because postCohort might add additional data parameters which the frontend can use (for example, "created_at" timestamps, default data if the post was missing some optional, data, and other things like that).
So something like
```
data: {
  cohort: rows[0],
  message: 'Cohort added successfully'
}
```